[2024-03-28T02:45:31.229+0000] {processor.py:153} INFO - Started process (PID=206) to work on /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:45:31.231+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py for tasks to queue
[2024-03-28T02:45:31.234+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:45:31.234+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:45:34.509+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:45:34.504+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py", line 15, in <module>
    from sklearn.utils.extmath import cartesian
ModuleNotFoundError: No module named 'sklearn'
[2024-03-28T02:45:34.513+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:45:34.771+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py took 3.676 seconds
[2024-03-28T02:46:36.210+0000] {processor.py:153} INFO - Started process (PID=433) to work on /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:46:36.212+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py for tasks to queue
[2024-03-28T02:46:36.214+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:46:36.213+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:46:39.520+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:46:39.501+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py", line 15, in <module>
    from sklearn.utils.extmath import cartesian
ModuleNotFoundError: No module named 'sklearn'
[2024-03-28T02:46:39.524+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:46:39.712+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py took 3.514 seconds
[2024-03-28T02:47:21.524+0000] {processor.py:153} INFO - Started process (PID=594) to work on /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:47:21.536+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py for tasks to queue
[2024-03-28T02:47:21.540+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:47:21.540+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:47:29.383+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:47:29.355+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py", line 15, in <module>
    from sklearn.utils.extmath import cartesian
ModuleNotFoundError: No module named 'sklearn'
[2024-03-28T02:47:29.399+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:47:29.513+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py took 8.021 seconds
[2024-03-28T02:48:12.289+0000] {processor.py:153} INFO - Started process (PID=810) to work on /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:48:12.294+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py for tasks to queue
[2024-03-28T02:48:12.298+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:48:12.298+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:48:16.293+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:48:16.283+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py", line 15, in <module>
    from sklearn.utils.extmath import cartesian
ModuleNotFoundError: No module named 'sklearn'
[2024-03-28T02:48:16.295+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:48:16.344+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py took 4.067 seconds
[2024-03-28T02:48:48.919+0000] {processor.py:153} INFO - Started process (PID=1014) to work on /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:48:48.922+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py for tasks to queue
[2024-03-28T02:48:48.924+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:48:48.923+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:48:50.943+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:48:50.939+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py", line 15, in <module>
    from sklearn.utils.extmath import cartesian
ModuleNotFoundError: No module named 'sklearn'
[2024-03-28T02:48:50.947+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:48:50.999+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py took 2.101 seconds
[2024-03-28T02:49:22.464+0000] {processor.py:153} INFO - Started process (PID=1220) to work on /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:49:22.466+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py for tasks to queue
[2024-03-28T02:49:22.470+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:49:22.470+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:49:24.228+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:49:24.213+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py", line 15, in <module>
    from sklearn.utils.extmath import cartesian
ModuleNotFoundError: No module named 'sklearn'
[2024-03-28T02:49:24.327+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:49:24.418+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py took 1.964 seconds
[2024-03-28T02:49:54.860+0000] {processor.py:153} INFO - Started process (PID=1426) to work on /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:49:54.865+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py for tasks to queue
[2024-03-28T02:49:54.867+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:49:54.867+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:49:55.902+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:49:55.898+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py", line 15, in <module>
    from sklearn.utils.extmath import cartesian
ModuleNotFoundError: No module named 'sklearn'
[2024-03-28T02:49:55.903+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:49:55.933+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py took 1.083 seconds
[2024-03-28T02:50:29.158+0000] {processor.py:153} INFO - Started process (PID=1635) to work on /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:50:29.165+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py for tasks to queue
[2024-03-28T02:50:29.168+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:50:29.167+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:50:31.508+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:50:31.495+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py", line 15, in <module>
    from sklearn.utils.extmath import cartesian
ModuleNotFoundError: No module named 'sklearn'
[2024-03-28T02:50:31.510+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:50:32.217+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py took 3.070 seconds
[2024-03-28T02:51:02.541+0000] {processor.py:153} INFO - Started process (PID=1848) to work on /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:51:02.546+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py for tasks to queue
[2024-03-28T02:51:02.549+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:51:02.548+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:51:03.782+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:51:03.777+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py", line 15, in <module>
    from sklearn.utils.extmath import cartesian
ModuleNotFoundError: No module named 'sklearn'
[2024-03-28T02:51:03.784+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:51:03.829+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py took 1.295 seconds
[2024-03-28T02:51:34.194+0000] {processor.py:153} INFO - Started process (PID=2059) to work on /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:51:34.197+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py for tasks to queue
[2024-03-28T02:51:34.199+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:51:34.199+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:51:35.068+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:51:35.066+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py", line 15, in <module>
    from sklearn.utils.extmath import cartesian
ModuleNotFoundError: No module named 'sklearn'
[2024-03-28T02:51:35.070+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:51:35.094+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py took 0.907 seconds
[2024-03-28T02:52:05.288+0000] {processor.py:153} INFO - Started process (PID=2268) to work on /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:52:05.291+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py for tasks to queue
[2024-03-28T02:52:05.293+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:52:05.292+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:52:06.772+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:52:06.768+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py", line 15, in <module>
    from sklearn.utils.extmath import cartesian
ModuleNotFoundError: No module named 'sklearn'
[2024-03-28T02:52:06.775+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:52:06.825+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py took 1.545 seconds
[2024-03-28T02:52:36.984+0000] {processor.py:153} INFO - Started process (PID=2467) to work on /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:52:36.986+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py for tasks to queue
[2024-03-28T02:52:36.988+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:52:36.987+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:52:38.040+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:52:38.037+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py", line 15, in <module>
    from sklearn.utils.extmath import cartesian
ModuleNotFoundError: No module named 'sklearn'
[2024-03-28T02:52:38.041+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:52:38.069+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py took 1.092 seconds
[2024-03-28T02:53:09.118+0000] {processor.py:153} INFO - Started process (PID=2672) to work on /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:53:09.134+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py for tasks to queue
[2024-03-28T02:53:09.136+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:53:09.136+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:53:11.733+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:53:11.720+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py", line 15, in <module>
    from sklearn.utils.extmath import cartesian
ModuleNotFoundError: No module named 'sklearn'
[2024-03-28T02:53:11.735+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:53:11.809+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py took 2.705 seconds
[2024-03-28T02:53:42.193+0000] {processor.py:153} INFO - Started process (PID=2885) to work on /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:53:42.195+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py for tasks to queue
[2024-03-28T02:53:42.199+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:53:42.198+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:53:43.077+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:53:43.074+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py", line 15, in <module>
    from sklearn.utils.extmath import cartesian
ModuleNotFoundError: No module named 'sklearn'
[2024-03-28T02:53:43.081+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:53:43.114+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py took 0.927 seconds
[2024-03-28T02:54:13.489+0000] {processor.py:153} INFO - Started process (PID=3096) to work on /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:54:13.491+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py for tasks to queue
[2024-03-28T02:54:13.492+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:54:13.492+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:54:14.440+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:54:14.438+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py", line 15, in <module>
    from sklearn.utils.extmath import cartesian
ModuleNotFoundError: No module named 'sklearn'
[2024-03-28T02:54:14.441+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:54:14.467+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py took 0.983 seconds
[2024-03-28T02:54:45.451+0000] {processor.py:153} INFO - Started process (PID=3318) to work on /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:54:45.452+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py for tasks to queue
[2024-03-28T02:54:45.454+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:54:45.453+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:54:46.228+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:54:46.226+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py", line 15, in <module>
    from sklearn.utils.extmath import cartesian
ModuleNotFoundError: No module named 'sklearn'
[2024-03-28T02:54:46.230+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:54:46.252+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py took 0.809 seconds
[2024-03-28T02:55:16.638+0000] {processor.py:153} INFO - Started process (PID=3529) to work on /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:55:16.639+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py for tasks to queue
[2024-03-28T02:55:16.640+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:55:16.640+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:55:17.482+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:55:17.478+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py", line 15, in <module>
    from sklearn.utils.extmath import cartesian
ModuleNotFoundError: No module named 'sklearn'
[2024-03-28T02:55:17.484+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:55:17.507+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py took 0.874 seconds
[2024-03-28T02:55:47.825+0000] {processor.py:153} INFO - Started process (PID=3739) to work on /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:55:47.827+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py for tasks to queue
[2024-03-28T02:55:47.828+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:55:47.827+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:55:48.768+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:55:48.759+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py", line 15, in <module>
    from sklearn.utils.extmath import cartesian
ModuleNotFoundError: No module named 'sklearn'
[2024-03-28T02:55:48.771+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:55:48.829+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py took 1.009 seconds
[2024-03-28T02:56:19.256+0000] {processor.py:153} INFO - Started process (PID=3950) to work on /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:56:19.258+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py for tasks to queue
[2024-03-28T02:56:19.259+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:56:19.258+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:56:20.299+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:56:20.294+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py", line 15, in <module>
    from sklearn.utils.extmath import cartesian
ModuleNotFoundError: No module named 'sklearn'
[2024-03-28T02:56:20.301+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:56:20.333+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py took 1.083 seconds
[2024-03-28T02:56:50.534+0000] {processor.py:153} INFO - Started process (PID=4161) to work on /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:56:50.536+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py for tasks to queue
[2024-03-28T02:56:50.537+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:56:50.537+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:56:51.389+0000] {logging_mixin.py:137} INFO - [2024-03-28T02:56:51.386+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py", line 15, in <module>
    from sklearn.utils.extmath import cartesian
ModuleNotFoundError: No module named 'sklearn'
[2024-03-28T02:56:51.391+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py
[2024-03-28T02:56:51.420+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/houzz_pro/dags/pro_schema/mobile_events.py took 0.893 seconds
